proxy/src/session.rs

```diff
@@ -222,4 +224,11 @@ pub mod settings {
             Self::Emulator
         }
     }
+
+    /// p2p config parameters subject to user preferences
+    #[derive(Debug, Default, Deserialize, Serialize, PartialEq)]
+    pub struct Ecosystem {
```

Comment:
We refer to the p2p domain as coco throughout the codebase, `Ecosystem` is quite general and could refer to many aspects.